docs/project_layout.md

```diff
@@ -0,0 +1,25 @@
+# Project Layout
+
+The hybrid helm project's scaffolding is customized to be compatible with both Helm and Go APIs.
+
+| File/Directory | Description | 
+| ------ | ----- |
+| `Dockerfile` | The Dockerfile of your operator project, used to build the image with `make docker-build`. |
+| `Makefile` | Build file with helper targets to help you work with your project. |
+| `PROJECT` | This file represents the project's configuration and is used to track useful information for the CLI and plugins. |
+| `bin/` | This directory contains useful binaries such as the `manager` which is used to run your project locally and  the `kustomize` utility used for the project configuration. For other language types, it might have other binaries useful for developing your operator. |
```

Comment:
We shouldn't have to explain that other language types. This section is specifically about the Hybrid Helm operator so you can safely focus on that part.

```suggestion
| `bin/` | This directory contains useful binaries such as the `manager` which is used to run your project locally and  the `kustomize` utility used for the project configuration. |
```